arch/xtensa/Kconfig

```diff
@@ -70,6 +70,12 @@ config XTENSA_SMALL_VECTOR_TABLE_ENTRY
 	  handlers to the end of vector table, renaming them to
 	  _Level\LVL\()VectorHelper.
 
+config XTENSA_RPO_CACHE
+	bool "Cached/uncached RPO mapping"
+	help
+	  Support Cached/uncached RPO mapping.
```

Comment:
Pedantically, can we move the comment in XTENSA_CACHED_REGION, which explains the trick being played, here instead?

Reply:
addressed